src/main/java/org/instituteatri/deep/controller/OccurrenceController.java

```diff
@@ -21,27 +26,132 @@ public class OccurrenceController {
+    )
+    @ApiResponses(value = {
+            @ApiResponse(
+                    responseCode = "200",
+                    description = "Success.",
+                    content = @Content(
+                            mediaType = "application/json",
+                            examples = @ExampleObject(
+                                    value = "[{" +
+                                            "\"id\": \"string\"," +
+                                            "\"name\": \"string\"," +
+                                            "\"description\": \"string\"" +
+                                            "}]"
+                            )
+                    )
+            )
+    })
     @GetMapping
     public ResponseEntity<List<OccurrenceResponseDTO>> getAll() {
         List<OccurrenceResponseDTO> all = service.getAll();
         LOGGER.info("Received request to get all occurrences");
         return ResponseEntity.ok().body(all);
     }
+    @Operation (
+            method = "GET",
+            summary = "Retrieve occurrence by ID",
+            description = "Returns the details of a specific occurrence identified by the provided ID."
+    )
+    @ApiResponses(value = {
+            @ApiResponse(responseCode = "200",
+                    description = "Success.",
+                    content = @Content(mediaType = "application/json",
+                            examples = @ExampleObject(
+                                    value = "{" +
+                                            " \"id\": \"e7b1a29f-4f5c-4c2d-b732-1b12f8e678a4\"," +
+                                            " \name\": \"string\", "+
+                                            " \"description\": \"string\""
+                            )
+                    )
+            ),
+            @ApiResponse(
+                    responseCode = "404",
+                    description = "Not Found.",
+                    content = @Content (mediaType = "application/json",
+                            examples = @ExampleObject(
+                                    value ="{\"message\":\"Could not find occurrence with id:e7b1a29f-4f5c-4c2d-b732-1b12f8e678a4\"}"
+                            ))
+            )
 
+    })
     @GetMapping("/{id}")
     public ResponseEntity<OccurrenceResponseDTO> getById(@PathVariable String id) {
         OccurrenceResponseDTO responseDTO = service.getById(id);
         LOGGER.info("Received request to get occurrence by ID");
         return ResponseEntity.ok().body(responseDTO);
     }
+    @Operation(
+            method = "POST",
+            summary = "Create a new occurrence.",
+            description = "Endpoint to create a new occurrence with the details provided in the request body. " +
+                    "Required JSON fields: 'name' (String) and 'description' (String). " +
+                    "Copy and use the following example JSON in the request body: " +
+                    " {" +
```

Comment:
You don't need this information, because the request body of the endpoint already contains the json schema that will be used to create an occurrence.
![p1](https://github.com/institute-atri/deep/assets/79126589/ef903c51-f415-49ad-a821-b9c5f4b91105)